test_runner/regress/test_storage_controller.py

```diff
@@ -719,6 +720,113 @@ def logged_stuck_again():
     env.storage_controller.consistency_check()
 
 
+def test_storage_controller_compute_hook_revert(
```

Comment:
nit: we can run this only on the default pg version

Reply:
https://github.com/neondatabase/neon/pull/9435/commits/96f28069c00016e8457fd8072216fcc363f20f1f